frontend/src/utils/utils.ts

```diff
@@ -138,3 +139,28 @@ export const getGasPrice = async (gasPriceConfidence: TxPriceConfidence = 99): P
     throw new Error(`Error fetching gas price from TxPrice API: ${message}`);
   }
 };
+
+export const isERC20 = async (address: string, provider: Provider) => {
+  const tokenContract = new Contract(address, ERC20_ABI, provider);
+  const hasDecimalsPromise = tokenContract.decimals();
+  const hasNamePromise = tokenContract.name();
+  const hasSymbolPromise = tokenContract.symbol();
+  try {
+    await Promise.all([hasSymbolPromise, hasNamePromise, hasDecimalsPromise]);
+    return true;
+  } catch {
+    return false;
+  }
+};
+
+export const isERC721 = async (address: string, provider: Provider) => {
+  const tokenContract = new Contract(address, ERC721_ABI, provider);
+  const hasNamePromise = tokenContract.name();
+  const hasSymbolPromise = tokenContract.symbol();
```

Comment:
We probably should look for a `tokenURI` to identify an NFT. I've seen contracts that have a `name` and `symbol` for other reasons, but aren't tokens

Reply:
That is a good point. The tricky thing with tokenURI is it requires an id argument. If we include it should we pass in an id of 1?

Reply:
Ahh, also a good point. Hm ok so wdyt of removing the ERC-20/721 distinction, looking for name and symbol, and if both exist say something like "this might be a token contract, such as ERC-20 or ERC-721". That way it's simpler, and other contracts that have name/symbol prob aren't ones you want to withdraw to anyway

Reply:
I think that is a great idea, I will remove the distinction.